projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -814,16 +875,21 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
                     .pipe(takeUntil(this.$destroy))
                     .subscribe(() => {
                         if (this.collapsed) {
-                            this.onTouchCallback();
+                            this.updateValidityOnBlur();
+                        }
+                        if (this.value && !this.value.start) {
+                            this.value = { start: null, end: null };
```

Comment:
Why is the value changing on the touch callback? Also if the goal is to clear the value.. do set a null TBH. That way a required validator can check it as well

Reply:
yeah with `null` is much better.
If one deletes start value and leave the component we should delete and the end value.
and this has nothing in common with touch callback, but the naming :)

Reply:
Yes, but you already handle the value change for those inputs, so I still don't see why the range adjusts values on blue as well.